src/main/java/uk/gov/hmcts/reform/civil/handler/callback/camunda/dashboardnotifications/claimant/TrialArrangementsNotifyOtherPartyNotificationHandler.java

```diff
@@ -0,0 +1,44 @@
+package uk.gov.hmcts.reform.civil.handler.callback.camunda.dashboardnotifications.claimant;
+
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.civil.callback.CallbackParams;
+import uk.gov.hmcts.reform.civil.callback.CaseEvent;
+import uk.gov.hmcts.reform.civil.callback.DashboardCallbackHandler;
+import uk.gov.hmcts.reform.civil.client.DashboardApiClient;
+import uk.gov.hmcts.reform.civil.model.CaseData;
+import uk.gov.hmcts.reform.civil.service.DashboardNotificationsParamsMapper;
+import uk.gov.hmcts.reform.civil.service.FeatureToggleService;
+
+import java.util.List;
+
+import static uk.gov.hmcts.reform.civil.callback.CaseEvent.CREATE_DASHBOARD_NOTIFICATION_TRIAL_ARRANGEMENTS_NOTIFY_CLAIMANT;
+import static uk.gov.hmcts.reform.civil.handler.callback.camunda.dashboardnotifications.DashboardScenarios.SCENARIO_AAA6_CP_TRIAL_ARRANGEMENTS_NOTIFY_OTHER_PARTY_CLAIMANT;
+
+@Service
+public class TrialArrangementsNotifyOtherPartyNotificationHandler extends DashboardCallbackHandler {
+
+    private static final List<CaseEvent> EVENTS =
+        List.of(CREATE_DASHBOARD_NOTIFICATION_TRIAL_ARRANGEMENTS_NOTIFY_CLAIMANT);
+    public static final String TASK_ID = "GenerateClaimantDashboardNotificationTrialArrangementsNotifyParty";
+
+    public TrialArrangementsNotifyOtherPartyNotificationHandler(DashboardApiClient dashboardApiClient,
+                                                    DashboardNotificationsParamsMapper mapper,
+                                                    FeatureToggleService featureToggleService) {
+        super(dashboardApiClient, mapper, featureToggleService);
+    }
+
+    @Override
+    public String camundaActivityId(CallbackParams callbackParams) {
+        return TASK_ID;
+    }
+
+    @Override
+    public List<CaseEvent> handledEvents() {
+        return EVENTS;
+    }
+
+    @Override
+    public String getScenario(CaseData caseData) {
```

Comment:
Do we need a check to see if the claimant is not represented? Otherwise we are saving notifications in the database that will never be seen. DashboardCallbackHandler does have a method that should allow you to present a condition: 
```
    @Override
    public boolean shouldRecordScenario(CaseData caseData) {
        return caseData.isRespondent1NotRepresented();
    }
```

Reply:
Good point @m-meulendijks-v1 . Updated now